Bio/SeqUtils/ProtParam.py

```diff
@@ -6,41 +6,40 @@
 """Simple protein analysis.
 
 Example:
-
-    >>> from Bio.SeqUtils.ProtParam import ProteinAnalysis
-    >>> X = ProteinAnalysis("MAEGEITTFTALTEKFNLPPGNYKKPKLLYCSNGGHFLRILPDGTVDGT"
-    ...                     "RDRSDQHIQLQLSAESVGEVYIKSTETGQYLAMDTSGLLYGSQTPSEEC"
-    ...                     "LFLERLEENHYNTYTSKKHAEKNWFVGLKKNGSCKRGPRTHYGQKAILF"
-    ...                     "LPLPV")
-    >>> print(X.count_amino_acids()['A'])
-    6
-    >>> print(X.count_amino_acids()['E'])
-    12
-    >>> print("%0.2f" % X.get_amino_acids_percent()['A'])
-    0.04
-    >>> print("%0.2f" % X.get_amino_acids_percent()['L'])
-    0.12
-    >>> print("%0.2f" % X.molecular_weight())
-    17103.16
-    >>> print("%0.2f" % X.aromaticity())
-    0.10
-    >>> print("%0.2f" % X.instability_index())
-    41.98
-    >>> print("%0.2f" % X.isoelectric_point())
-    7.72
-    >>> sec_struc = X.secondary_structure_fraction()  # [helix, turn, sheet]
-    >>> print("%0.2f" % sec_struc[0])  # helix
-    0.28
-    >>> epsilon_prot = X.molar_extinction_coefficient()  # [reduced, oxidized]
-    >>> print(epsilon_prot[0])  # with reduced cysteines
-    17420
-    >>> print(epsilon_prot[1])  # with disulfid bridges
-    17545
-
-    Other public methods are:
-     - gravy
-     - protein_scale
-     - flexibility
+>>> from Bio.SeqUtils.ProtParam import ProteinAnalysis
```

Comment:
Does this render correctly? Do we need something like:

```rst
Examples
--------
>>> print(123)
123
```

Note this is deliberately with a "s" and no colon as in many other places in Biopython - to match the numpydoc standard, which seems the most likely standard we might want to adopt.

Reply:
I thought that the `>>>` should be enough to mark this as code block, unfortunately, you seem to be right that it is not rendered correctly (at least when relying on https://livesphinx.herokuapp.com/ or http://rst.aaroniles.net/, although I had the feeling in the past that even rst.ninjs.org was not doing everything correctly. Unfortunately, I don't have an actual example)

Do you have an example how 
```
Examples
----------
>>> print(123)
123
```
is rendered in Sphinx? Regarding the size of the header? If it is rendered as larger header, I'd prefer to mark the code block by a simple one-space indentation.

Reply:
Looks OK here, really need an example with more than one section...

https://github.com/biopython/biopython/blob/biopython-174/Bio/File.py#L78
https://biopython.org/docs/1.74/api/Bio.File.html#Bio.File.as_handle
https://biopython.org/DIST/docs/api/Bio.File-module.html#as_handle

Reply:
Okay, I go with this. 

Can you clarify what you mean with 'check the RST with a doctest example' below?

Reply:
It is the same situation - rather than this:

```
Example:
>>> regex('ABCGN')
'A[CGT]CG.'
```

How about:

```
Examples
--------
>>> regex('ABCGN')
'A[CGT]CG.'
```